src/platform/forms-system/src/js/utilities/ui/index.js

```diff
@@ -6,20 +6,15 @@ export function focusElement(selectorOrElement, options) {
     typeof selectorOrElement === 'string'
       ? document.querySelector(selectorOrElement)
       : selectorOrElement;
-  let prevTabIdx = null;
 
   if (el) {
-    prevTabIdx = el.getAttribute('tabindex');
-    if (prevTabIdx && prevTabIdx <= 0) {
+    if (el.tabIndex === 0) {
```

Comment:
Honestly, I don't know why I have to do this. IE seems to treat some non-focusable elements has having tabindex of 0.